composition/src/schema-building/ast.ts

```diff
@@ -217,7 +232,7 @@ export type MutableNonNullTypeNode = {
 };
 
 // TODO errors
```

Comment:
Whats the exact todo here?

Reply:
I forgot to replace the error. Thanks for pointing it out—done.